Python/syntax_test_python.py

```diff
@@ -1519,51 +1519,46 @@ class Starship:
 #                                                      ^^^^^ comment.line.number-sign - comment.line.type-hint
 
 # With/for allow commas
-for a, b in lst: # type: str, int
-#                           ^ comment.line.type-hint punctuation.separator.sequence
+for a, b in lst: # type: (str, int)
```

Comment:
The parens in `(str, int)`are not allowed.  It's actually `Tuple[str, int]`.  They take explicit > implicit seriously.

Reply:
I am uncertain here. I've created a little test file with

```python
from typing import Tuple

def testing(results):
    for a, b in results:  # type: Tuple[int, str]
        print(a, b)
```

Hovering `a` or `b` with LSP and LSP-pylsp installed, doesn't show anything, while an example using current syntax with or without parentheses works fine.

```python
def testing(results):
    for a, b in results:  # type: (int, str)
        print(a, b)
```

It also makes sense as I found some comments about such type hints to date back to python 2.7 which doesn't know about `typing` module.

Reply:
The related spec can be found at https://mypy.readthedocs.io/en/stable/type_inference_and_annotations.html#declaring-multiple-variable-types-at-a-time